test-configs.yaml

```diff
@@ -600,6 +630,16 @@ device_types:
             - 'multi_v7_defconfig+kselftest'
           kernel: ['v3.18']
 
+  imx6q-sabresd:
+    mach: freescale
+    class: arm-dtb
+    boot_method: uboot
+
+  imx7d-sdb:
+    mach: freescale
+    class: arm-dtb
+    boot_method: uboot
```

Comment:
Baseline results on staging: https://staging.kernelci.org/test/plan/id/5e9fa00a0a7ee77c3661e038/
All passing.

Upstream in LAVA:
https://git.lavasoftware.org/lava/lava/-/blob/master/etc/dispatcher-config/device-types/imx7d-sdb.jinja2